schema/traces/README.md

```diff
@@ -0,0 +1,104 @@
+
+- **type** conforms to one of the supported Observability signals (Traces, Logs, Metrics, Alerts)
+- **dataset** user defined field that can mainly be utilized for describing the origin of the signal
+- **namespace** user custom field that can be used to describe any customer domain specific classification
+
+
+If nothing is stated in the namespace / dataset - the signal information would be routed into the default data-stream indices
+- **Traces** - traces-default-namespace
+- **Metrics** - metrics-default-namespace
+- **Logs** -   logs-default-namespace
+
+#### Timestamp field
+As part of the data-stream definition the `@timestamp` is  mandatory, if the field is not present to begin with use `ObservedTimestamp` as value for this field
+
+### Instrumentation scope
+This is a logical unit of the application with which the emitted telemetry can be associated. It is typically the developer’s choice to decide what denotes a reasonable instrumentation scope.
+The most common approach is to use the instrumentation library as the scope, however other scopes are also common, e.g. a module, a package, or a class can be chosen as the instrumentation scope.
+
+The instrumentation scope may have zero or more additional attributes that provide additional information about the scope. As an example the field
+`instrumentationScope.attributes.identification` is presented will be used to determine the resource origin of the signal and can be used to filter accordingly
+
+## Traces
+see [OTEL traces convention](https://github.com/open-telemetry/opentelemetry-specification/tree/main/semantic_conventions/trace)
+
+Traces in are defined implicitly by their Spans - In particular, a Trace can be thought of as a directed acyclic graph (DAG) of Spans, where the edges between Spans are defined as parent/child relationship.
+
+## Spans
+A span represents an operation within a transaction. Each Span encapsulates the following state:
+
+* An operation name
+* start and finish timestamp
+* Attributes list of key-value pairs.
+* Set of Events, each of which is itself a tuple (timestamp, name, Attributes)
+* Parent's Span identifier.
+* Links to causally-related Spans (via the SpanContext of those related Spans).
+* SpanContext information required to reference a Span. 
+
+### SpanContext
+Represents all the information that identifies Span in the Trace and is propagated to child Spans and across process boundaries.
+A **SpanContext** contains the tracing identifiers and the options that are propagated from parent to child Spans.
+
+* TraceId  - It is worldwide unique with practically sufficient probability by being made as 16 randomly generated bytes - used to group all spans for a specific trace together across all processes. 
+* SpanId  -  It is the identifier for a span, globally unique with practically sufficient probability by being made as 8 randomly generated bytes. When passed to a child Span this identifier becomes the parent span id for the child Span.
```

Comment:
Do we need different randomness entropies? Does a simple `UUID` not suffice for both?

Reply:
this is the OTEL spec - we just publish the spec as is...